<launch>
    <arg name="camera_name" default="camera" />
    <arg name="device_id" default="0" />
    <arg name="device_ip" default="192.168.1.101" />
    <arg name="calib_file" default="" />
    
        <node   name="$(arg camera_name)"
                pkg="fast_cam"
                type="camera_.py"
                output="screen">
            <param name="device_id" value="$(arg device_id)" />
            <param name="device_ip" value="$(arg device_ip)" />
            <param name="camera_manager" value="$(arg camera_name)" />
            <param name="calibration_file" value="$(find fast_cam)/config/camera_info_$(arg calib_file).yaml"/>
    </node>
</launch>
